### PR TITLE
Github Actions and Serverless Application Repository

### PR DESCRIPTION
**Pull request by matt-tyler**

### CODE REVIEW

**Comment by temyers** [comment]
Looking good so far.

**Comment by matt-tyler** [comment]
Adding some images and changed the date - ready to merge in a fortnight

File: _posts/2019-12-25-github-actions-for-sam.md [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+
+# My Typical Release Workflow
+
+My typical CI pattern for something contains the following elements;
+
+1. A continuous integration process that builds and tests every push/merge to every branch.
+
+This is to ensure that the branch can be built and tested correctly before pushed to master. If this fails, I want to prevent the code from being merged to master. Once merged, the state of the master branch has changed and therefore I want the tests to be re-executed to ensure the head of master remains in a releasable state.
+
+2. A process that releases an artifact when a tag is pushed to the head of master.
```
Comment (t04glovern) [inline _posts/2019-12-25-github-actions-for-sam.md]: The numbering when it actually renders doesn't work. Might be better to use dot points instead

image

Comment (matt-tyler) [reply _posts/2019-12-25-github-actions-for-sam.md]: I've fixed it. Needed to indent the paragraphs under the list in markdown to get the formatting right.

**Pull request merged**